addons/website/views/snippets/snippets.xml

```diff
@@ -1414,16 +1428,13 @@
                   data-dependencies="!header_effect_scroll_opt"/>
     </div>
 
-    <!-- Footer - Colors & Layouts -->
-    <div data-js="WebsiteLevelColor"
-         data-selector="#wrapwrap > footer"
+        <!-- Footer - Layouts -->
+    <div data-selector="#wrapwrap > footer"
          data-no-check="true"
          groups="website.group_website_designer">
-
-        <!-- Layouts -->
         <we-select string="Template"
-            data-variable="footer-template"
-            data-reload="/">
+                   data-variable="footer-template"
+                   data-reload="/">
```

Comment:
Fair enough, but try to avoid introducing unnecessary diffs, such as those caused by linting or indentation changes.